virttest/qemu_devices/qcontainer.py

```diff
@@ -2018,7 +2018,8 @@ def images_define_by_params(self, name, image_params, media=None,
 
     def serials_define_by_variables(self, serial_id, serial_type, chardev_id,
                                     bus_type=None, serial_name=None,
-                                    bus=None, nr=None, reg=None):
+                                    bus=None, nr=None, reg=None,
+                                    pci_extra_params=None):
```

Comment:
It looks this var should be bus_extra_params.

Reply:
Thanks and update.